api/src/main/java/io/strimzi/api/kafka/model/KafkaBridgeSpec.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.KafkaBridgeTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "replicas", "image", "http", "kafka", "consumer",
+        "producer", "resources", "jvmOptions", "logging",
+        "metrics", "livenessProbe", "readinessProbe", "template"})
+@EqualsAndHashCode
+public class KafkaBridgeSpec implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private int replicas;
+
+    private String image;
+    private KafkaBridgeHttpConfig http;
+    private KafkaBridgeConfigurationSpec kafka;
```

Comment:
I'm not sure I understand why are these things wrapped in the `kafka` field. Should they not be directly in `KafkaBridge.spec` instead of `KafkaBridge.spec.kafka`? This is a KafkaBridge, so it is obvious that these fields will configure Kafka.

Reply:
Yeah it's something like Kafka MM, where consumer and producer aren't under a `kafka` field. I would agree.